config/jobs/kubernetes/sig-cloud-provider/azure/release-1.22.yaml

```diff
@@ -8,7 +8,7 @@ presubmits:
     run_if_changed: 'azure.*\.go'
     path_alias: k8s.io/kubernetes
     branches:
-      - release-1.18
+      - release-1.22
```

Comment:
we don't need to update generate.sh?

Reply:
We don't need to because the script takes a list of versions as arguments and generate a YAML file for each version (i.e. `./config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh 1.19 1.20 1.21 1.22 master`.